crates/re_entity_db/src/entity_db.rs

```diff
@@ -215,7 +215,8 @@ impl EntityDb {
     }
 
     pub fn num_rows(&self) -> usize {
-        self.data_store.num_timeless_rows() as usize + self.data_store.num_temporal_rows() as usize
+        (self.data_store.num_static_cells() > 0) as usize
```

Comment:
this should have a comment on why all static data is considered as one row

Reply:
uh oh this is wrong actually